docs/content/assets/css/customizations.css

```diff
@@ -0,0 +1,20 @@
+/* This file adds our styling additions to maintain. */
+
+/* External Link icon feature. Rejected from upstreaming to `mkdocs-material`.
+Alternative solution using SVG icon here (Broken on Chrome?): https://github.com/squidfunk/mkdocs-material/issues/2318#issuecomment-789461149
+Tab or Nav sidebar with non-relative links will prepend an icon (font glyph)
+If you want to append instead, switch `::before` to `::after`.
+*/
+/* reference the icon font to use */
+@font-face {
+  font-family: 'external-link';
+  src: url('/docker-mailserver/assets/fonts/external-link.woff') format('woff');
+}
+
+/* Matches the two nav link classes that start with `http` `href` values, regular docs pages use relative URLs instead. */
+.md-tabs__link[href^="http"]::before, .md-nav__link[href^="http"]::before {
+  display: inline-block; /* treat similar to text */
+  font-family: 'external-link';
+  content:'\0041'; /* represents "A" which our font renders as an icon instead of the "A" glyph */
+  font-size: 80%; /* icon is a little too big by default, scale it down */
+}
```

Comment:
I really like the comments you add. This enables everyone to quickly follow up.

Reply:
Thanks :)

I realize I might not be around to help with maintenance in future and that not many here may be comfortable with CSS so I've tried to lighten that burden.